ash/src/extensions/khr/external_memory_win32.rs

```diff
@@ -0,0 +1,63 @@
+use crate::prelude::*;
+use crate::vk;
+use crate::{Device, Instance};
+use std::ffi::CStr;
+use std::mem;
+use std::ptr;
+
+/// <https://www.khronos.org/registry/vulkan/specs/1.3-extensions/man/html/VK_KHR_external_memory_win32.html>
+#[derive(Clone)]
+pub struct ExternalMemoryWin32 {
+    handle: vk::Device,
+    fp: vk::KhrExternalMemoryWin32Fn,
+}
+
+impl ExternalMemoryWin32 {
+    pub fn new(instance: &Instance, device: &Device) -> Self {
+        let handle = device.handle();
+        let fp = vk::KhrExternalMemoryWin32Fn::load(|name| unsafe {
+            mem::transmute(instance.get_device_proc_addr(handle, name.as_ptr()))
+        });
+        Self { handle, fp }
+    }
+
+    /// <https://www.khronos.org/registry/vulkan/specs/1.3-extensions/man/html/vkGetMemoryWin32HandleKHR.html>
+    pub unsafe fn get_memory_win32_handle(
+        &self,
+        create_info: &vk::MemoryGetWin32HandleInfoKHR,
+    ) -> VkResult<vk::HANDLE> {
+        let mut handle = ptr::null_mut();
```

Comment:
Fwiw this should probably be `INVALID_HANDLE_VALUE` or `-1`?

Reply:
Anyone opposed to me adding a `pub const INVALID_HANDLE_VALUE: HANDLE = -1isize as *mut c_void;` to `platform_types.rs`?

Reply:
In what circumstance is the initial value here observable?

Reply:
The Vulkan function could observe it (even though it shouldn't). Theoretically it could also not update it while returning `vk::Result::SUCCESS`.

Probably chasing ghosts here.

Reply:
I think that would constitute a violation of the spec.

Reply:
Fair enough. Let's get it merged then.